src/babel/__tests__/__fixtures__/server/implicit-no-ssr/output.js

```diff
@@ -1,4 +1,4 @@
 import { lazy } from 'react-loosely-lazy';
-const ImplicitNoSsr = lazy(() => import('react'), {
+const ImplicitNoSsr = lazy(() => () => null, {
```

Comment:
What happens if somebody writes `import('react').then(...)` ? Or `lazy(() => markBundleReady(import('react')))` ?
Should we replace with something more promise like or is that already taken care of by a later transformation?

Reply:
But if SSR is set to false and we're on the server, why should the `.then` execute?

Reply:
Right, then I think it's fine. Can we just add a test to ensure that the generated js by babel is fine? Not sure if we might end up with `() => null.then(...)` (should not but better cover the scenario)

Reply:
Yep, that was a good test to add since it had the bad output you mentioned 😢 I have added the tests and fixed the logic now.